Storage/Processor/Image.php

```diff
@@ -47,7 +47,7 @@ public function generate(string $filename)
         }
 
         $path = tempnam(sys_get_temp_dir(), 'ems_image');
-        if ($this->config->getQuality()) {
+        if (($this->config->getQuality() ?? 0)  > 0) {
```

Comment:
I think the same goes for quality? Should return 0 by default. (I don’t really like this combination of ?? Inside an if test. This is making 0 a default value so probably getQuality should by default return 0? If not it should by default generate 100, but not an incompatible value (null)

Reply:
Fixed